lib3h_protocol/src/data_types.rs

```diff
@@ -0,0 +1,195 @@
+use crate::Address;
+
+/// Tuple holding all the info required for identifying a metadata.
+/// (entry_address, attribute, content)
+/// TODO: Figure out if we keep this
+pub type MetaTuple = (Address, String, Vec<u8>);
```

Comment:
Just wondering why using a tuple here instead of a `named-field struct`, wouldn't it be easier to use later on and less prone to error if one day we decide to add some element to this data structure ? Furthermore, a `named-field struct` will actually carry out the purpose of its content within it so it will be easier to understand what this data structure do.
What do you think ?

Reply:
No need for a struct for its current use, and this might get removed in the future so I would like to keep it as simple as possible.